microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -139,7 +150,9 @@ def _install(self) -> bool:
             mkdir.create_directory(self._remote_tar_path.parent.as_posix())
             self.node.shell.copy(PurePath(self._tar_file_path), self._remote_tar_path)
             self.node.tools[Tar].extract(
-                str(self._remote_tar_path), str(self._installed_path)
+                str(self._remote_tar_path),
+                str(self._installed_path),
+                sudo=self._install_as_sudo,
```

Comment:
Does it need? The original code doesn't use sudo, but it works. It doesn't need to add `sudo` for this command.